stable/distribution/CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # JFrog Distribution Chart Changelog
 All changes to this project chart be documented in this file.
 
-## [3.2.1] - Mar 13, 2019
-* Move securityContext to container level
+## [3.2.2] - Mar 15, 2019
```

Comment:
@danielezer  We need 3.2.1's change log as well

Reply:
@jainishshah17 Done, thanks.